### PR TITLE
overlay: PL2: Add overlays for PL2

### PR DESCRIPTION
**Pull request by theimpulson**

### CODE REVIEW

File: Nokia/PL2/res/values/config.xml [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+    <!-- Values for GPS configuration -->
+    <string-array translatable="false" name="config_gpsParameters">
+        <item>SUPL_HOST=supl.google.com</item>
+        <item>SUPL_PORT=7275</item>
+        <item>XTRA_SERVER_1=http://xtrapath1.izatcloud.net/xtra3grc.bin</item>
+        <item>XTRA_SERVER_2=http://xtrapath2.izatcloud.net/xtra3grc.bin</item>
+        <item>XTRA_SERVER_3=http://xtrapath3.izatcloud.net/xtra3grc.bin</item>
+        <item>NTP_SERVER=north-america.pool.ntp.org</item>
+        <item>SUPL_MODE=1</item>
+        <item>SUPL_VER=0x20000</item>
```
Comment (phhusson) [inline Nokia/PL2/res/values/config.xml:309]: You should probably look at switching to version 3 for security reasons

File: Nokia/PL2/res/values/config.xml [hunk 2]
```diff
@@ -0,0 +1,331 @@
...
+    <bool name="config_dozePulsePickup">true</bool>
+
+    <!-- Specifies whether to decouple the interactive state of the device from the display on/off state. -->
+    <bool name="config_powerDecoupleInteractiveModeFromDisplay">true</bool>
+
+    <!-- Is the device capable of hot swapping an UICC Card -->
+    <bool name="config_hotswapCapable">true</bool>
+
+    <!-- Flag specifying whether VoLTE is available on device -->
+    <bool name="config_device_vt_available">true</bool>
```
[the remark this answers is not in the document]
Comment (theimpulson) [reply Nokia/PL2/res/values/config.xml]: Removed

File: Nokia/PL2/res/values/config.xml [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+** limitations under the License.
+*/
+-->
+
+<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
+
+    <!-- Indicate whether closing the lid causes the device to go to sleep and opening
+         it causes the device to wake up.
+         The default is false. -->
+    <bool name="config_lidControlsSleep">true</bool>
```
[the remark this answers is not in the document]
Comment (theimpulson) [reply Nokia/PL2/res/values/config.xml]: My bad, removed

File: Nokia/PL2/res/values/config.xml [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+
+    <!-- Wifi driver supports batched scan -->
+    <bool translatable="false" name="config_wifi_batched_scan_supported">true</bool>
+
+    <!-- If true, the doze component is not started until after the screen has been turned off
+         and the screen off animation has been performed. -->
+    <bool name="config_dozeAfterScreenOffByDefault">true</bool>
+
+    <!-- Doze: should the TYPE_PICK_UP_GESTURE sensor be used as a pulse signal. -->
+    <bool name="config_dozePulsePickup">true</bool>
```
[the remark this answers is not in the document]
Comment (theimpulson) [reply Nokia/PL2/res/values/config.xml]: Here you go: https://paste.ubuntu.com/p/vCwkbQPg7H/

File: Nokia/PL2/res/values/config.xml (hunk 1 above)
Comment (theimpulson) [reply Nokia/PL2/res/values/config.xml]: Any specific reason? I can't see any other device doing that, so curious.

File: Nokia/PL2/res/values/config.xml (hunk 3 above)
[the remark this answers is not in the document]
Comment (phhusson) [reply Nokia/PL2/res/values/config.xml]: Are you sure?
That was a genuine question. A "lid" can be a smartcover for instance

[the remark this answers is not in the document]
Comment (theimpulson) [reply Nokia/PL2/res/values/config.xml]: Yep, sure. It doesn't have a lid.

File: Nokia/PL2/res/values/config.xml (hunk 1 above)
Comment (phhusson) [reply Nokia/PL2/res/values/config.xml]: I can't seem to find again my reference, and I'd like to have it before having actually you do it, so maybe don't...

Basically, with SUPL_VER 2.0, the blobs you download from XTRA server are not signed, or anything, and GPS software often have various RCEs on the parsing of those blobs, so signing them protect against such RCEs (you'll notice that XTRA servers are http not https)

**Pull request merged**